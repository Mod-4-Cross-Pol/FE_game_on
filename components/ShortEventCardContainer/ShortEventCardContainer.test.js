import React from 'react';
// import { ShortEventCardContainer } from './ShortEventCardContainer';
import renderer from 'react-test-renderer';

describe('ShortEventCardContainer', () => {

  describe('Renders a ShortEventCardContainer component', () => {

    // test('renders correctly', () => {
    //   const wrapper = renderer.create(<ShortEventCardContainer />).toJSON();
    //   expect(wrapper).toMatchSnapshot();
    // });

    test('it works', () => {
      expect(true).toBeTruthy();
    });

  });

});
