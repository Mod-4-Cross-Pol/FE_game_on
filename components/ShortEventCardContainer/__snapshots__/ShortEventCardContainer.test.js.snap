// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShortEventCardContainer Renders a ShortEventCardContainer component renders the component 1`] = `ShallowWrapper {}`;
